src/VisualStudio/Core/Impl/CodeModel/InternalElements/CodeParameter.cs

```diff
@@ -62,6 +62,17 @@ protected override EnvDTE.CodeElements GetCollection()
             return GetCollection<CodeParameter>(Parent);
         }
 
+        protected override string GetName()
+        {
+            return this._name;
```

Comment:
Nitpick: we probably don't need the this. anymore given we're doing _ qualification.